features/signal/child_workflow/feature.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from temporalio import workflow
+from temporalio.client import WorkflowHandle
+
+from harness.python.feature import Runner, register_feature
+
+CHILD_WORKFLOW_INPUT = "child-input"
+CHILD_WORKFLOW_INITIAL_STATE = "1"
+SIGNAL_ARG = "2"
```

Comment:
Don't have to change, but for me extract constants out in these tiny tests doesn't have much value

Reply:
Agree I had too many constants. I think I still like one or two of them where they are repeated several times and critical to test logic.